### PR TITLE
Update LegitimateURLShortener.txt

### PR DESCRIPTION
**Pull request by Nojuuu**
Cleans `etsy.com` URLs.

### CODE REVIEW

File: LegitimateURLShortener.txt [hunk 1]
```diff
@@ -1847,6 +1847,14 @@ $removeparam=iv_,domain=allbirds.com
 ||weidian.com^$removeparam=ifr
 ||weidian.com^$removeparam=share_relation
 
+! https://www.etsy.com/listing/655283263/docking-stationgift-for-men-for?organic_search_click=1&pop=1
+! https://www.etsy.com/listing/984664013/personalized-leather-walletmens?plkey=e61127eaa78c1a3d517e415d3e2014db7e43e6f5%3A984664013&pro=1&frs=1
+$removeparam=organic_search_click,domain=etsy.com
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:1852]:
```suggestion
$removeparam=organic_search_click,domain=etsy.*
```
Probably this should use a wildcard TLD as to work for other countries.

File: LegitimateURLShortener.txt [hunk 2]
```diff
@@ -1847,6 +1847,14 @@ $removeparam=iv_,domain=allbirds.com
...
+$removeparam=pop,domain=etsy.com
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:1853]: This seems like it could break something.

**Review by iam-py-test — COMMENTED** [review]
This looks ok, but there are a few things I'm not sure about

File: LegitimateURLShortener.txt (hunk 2 above)
Comment (Nojuuu) [reply LegitimateURLShortener.txt:1853]: I haven't noticed any breakage and someone can send you link without those parameters anyway so i'd assume that the site should work fine without those.

Comment (iam-py-test) [reply LegitimateURLShortener.txt:1853]: Ok. Thanks!

File: LegitimateURLShortener.txt (hunk 1 above)
Comment (Nojuuu) [reply LegitimateURLShortener.txt:1852]: I checked, and the addresses for different countries seem to be like that:
```
https://www.etsy.com/au/
https://www.etsy.com/ca/
https://www.etsy.com/no-en/
https://www.etsy.com/se-en/
....
```

Comment (iam-py-test) [reply LegitimateURLShortener.txt:1852]: I did find one on another TLD but it just redirected

**Comment by iam-py-test** [comment]
Thank you for opening this Pull Request. Merging

**Pull request merged**